.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install cookiecutter
+        run: sudo apt-get install cookiecutter
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15.x
```

Comment:
Since Go 1.16 has just been released, should we test for it too? How about using matrix testing for the last 2 versions -> 1.15 and 1.16?

Reply:
That is a good idea but how about to use Go 1.16 in our template instead of 1.15 and update the **Prerequisite** in the README? So we don't need to test for the last 2 versions.

Reply:
Sure, we can do that as well for now. I just think that because of the short release cycle of Go, it could be a nice-to-have to test for the current and previous version.